variables.tf

```diff
@@ -72,19 +79,74 @@ variable "cloud_connection_speed" {
 #####################################################
 
 variable "tags" {
-  description = "List of Tag names for IBM Cloud PowerVS service"
+  description = "List of tag names for the IBM Cloud PowerVS service"
   type        = list(string)
   default     = null
 }
 
 variable "cloud_connection_gr" {
-  description = "Enable global routing for this cloud connection.Can be specified when creating new connection"
+  description = "Whether to enable global routing for this IBM Cloud connection. You can specify thia value when you create a connection."
   type        = bool
   default     = null
 }
 
 variable "cloud_connection_metered" {
-  description = "Enable metered for this cloud connection. Can be specified when creating new connection"
+  description = "Whether to enable metering for this IBM Cloud connection. You can specify thia value when you create a connection."
   type        = bool
   default     = null
 }
+
+variable "access_host_or_ip" {
```

Comment:
There should be a default here set to null - otherwise this forces users of the module to have a jump box even if they opt out of deployment the power managemenr services.

Reply:
the only scenario where access host is not needed is basic. Otherwise, creating landscape that you cannot reach, does not make sense in my oppinion

Reply:
Added